src/components/carousel/_index.scss

```
@@ -0,0 +1,54 @@
+@use "../../scss";
+
+.c-carousel {
+	--slide-width: var(--carousel-slide-width, 25%);
+
+	display: grid;
+	grid-template-areas: "carousel";
+	justify-content: center;
+	overflow: hidden;
+
+	@include scss.component-properties("carousel");
+
+	> * {
+		grid-area: carousel;
+	}
+
+	&__track {
+		display: flex;
+		transition: transform 0.4s ease-in;
+		will-change: transform;
+
```

Part 2 of touch fix:
```suggestion
		will-change: transform;
		touch-action: 'pan-y';

```

---

ohh interesting, haven't seen `pan-y` before

---

Stole it from here... 

https://github.com/FormidableLabs/react-swipeable#how-to-use-touch-action-to-prevent-scrolling

...but now that I'm working with it, it doesn't seem to do anything. :P